### PR TITLE
Add session header to response

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

**Comment by yohanboniface** [comment]
Goal is to make this information available to Nginx, so we can log, and then use it in ELK.

@davidbgk @vinyll r?

File: ban/tests/http/test_municipality.py [hunk 1]
```diff
@@ -435,3 +435,13 @@ def test_cannot_change_deleted_at_with_put(put):
     assert resp.status_code == 200
     assert not models.Municipality.get(
         models.Municipality.pk == municipality.pk).deleted_at
+
+
+@authorize
```
Comment (vinyll) [inline ban/tests/http/test_municipality.py]: No import `authorize`?

File: ban/tests/http/test_municipality.py [hunk 2]
```diff
@@ -435,3 +435,13 @@ def test_cannot_change_deleted_at_with_put(put):
...
+def test_authorized_responses_contain_sessions_data(get):
+    municipality = MunicipalityFactory(name="Cabour")
+    resp = get('/municipality/{}'.format(municipality.id))
+    assert resp.status_code == 200
+    session = context.get('session')
+    assert resp.headers['Session-Client'] == session.client.id
```
Comment (vinyll) [inline ban/tests/http/test_municipality.py]: I can't find out where the `session.client` and `session.user` got injected in this test.

File: ban/http/auth.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
+from functools import wraps
+
 from .utils import abort
 from flask_oauthlib.provider import OAuth2Provider
-from flask import request
+from flask import request, make_response
```
Comment (yohanboniface) [inline ban/http/auth.py]: Old suff, good catch :)

File: ban/tests/http/test_municipality.py (hunk 1 above)
Comment (yohanboniface) [inline ban/tests/http/test_municipality.py]: It's imported from before :)

File: ban/tests/http/test_municipality.py (hunk 2 above)
Comment (yohanboniface) [inline ban/tests/http/test_municipality.py]: This is not specific to this test: the session is saved when authenticating the request:

https://github.com/BaseAdresseNationale/ban/blob/master/ban/http/auth.py#L44

**Comment by yohanboniface** [comment]
Thanks :)

**Pull request merged**